### PR TITLE
Fix Invoice PDF title

### PR DESCRIPTION
**Pull request by marcogbarcellos**
Increasing line height and fixing format to `A4` as `Letter` format is exceeding the pdf standard width in some cases.
 
fixes https://github.com/opencollective/opencollective/issues/1374

### CODE REVIEW

File: src/server/controllers/transactions.js [hunk 1]
```diff
@@ -34,10 +34,7 @@ export async function invoice(req, res, next) {
     return next(e);
   }
 
-  const pageFormat =
-    req.query.pageFormat === 'A4' || invoice.fromCollective.currency === 'EUR'
-      ? 'A4'
-      : 'Letter';
+  const pageFormat ='A4';
```
Comment (xdamman) [inline src/server/controllers/transactions.js]: Why?

Format should be different for Europe (A4) and US (Letter).

Comment (marcogbarcellos) [reply src/server/controllers/transactions.js]: Hum, I see. the Letter format is going over the pdf width limit, looks buggy to me but I'll double check and look for a better workaround.

**Comment by marcogbarcellos** [comment]
@xdamman this is what I meant:

- Letter:
screen shot 2018-11-03 at 6 50 40 am

- A4:
screen shot 2018-11-03 at 6 51 12 am

**Comment by marcogbarcellos** [comment]
Fixed it decreasing the width of the letter page format and adding a left margin on it: 
screen shot 2018-11-03 at 7 17 55 am

File: src/pages/invoice.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class InvoicePage extends React.Component {
       Letter: {
         unit: 'in',
         page: {
-          width: 8.5,
+          width: 7.5,
```
Comment (xdamman) [inline src/pages/invoice.js]: This is a hack that will create confusion.
The proper dimensions of the Letter format is 8.5 x 11 https://en.wikipedia.org/wiki/Letter_%28paper_size%29
Let's respect conventions here. You can play with margins, paddings, etc if needed.

**Pull request merged**